### DISCUSSION TITLE
Color scale for electric field results

### DISCUSSION THREAD

**Discussion by xiberJ** [simnibs/simnibs#226] [9. Results and analysis]
Dear developer, hello.
I would like to ask whether the visualization of electric field results can fix the scale of its electric field color, so as to compare the difference between the two simulation results, because the deepest color is the maximum electric field, and when comparing the two result graphs, the maximum electric field is different, so it is not easy to distinguish

**Comment by oulap** [comment]
Please see here how to change the e-field visualization ranges:

https://simnibs.github.io/simnibs/build/html/tutorial/visualization.html
